Formula/p/pyenv.rb

```diff
@@ -1,9 +1,11 @@
 class Pyenv < Formula
   desc "Python version management"
   homepage "https://github.com/pyenv/pyenv"
-  url "https://github.com/pyenv/pyenv/archive/refs/tags/v2.4.5.tar.gz"
-  sha256 "24f3671a92492f49fa6f61dea861323ad853877c5cb2686c95bfa76339aa1301"
+  url "https://github.com/pyenv/pyenv/archive/2c2c7616e2c3a8ab7623bea3c353ff520750b812.tar.gz"
+  version "2.4.5"
+  sha256 "878e734212f284e06323ac0f2e3c7453f597eb655ced7c8a608bbb20448a70c9"
   license "MIT"
+  revision 1
```

Comment:
no need to bump (unless the previous tag had some big issue, it would just rebuild the current artifact)

Reply:
As per above, there is "some big issue".